README.md

```diff
@@ -16,15 +16,15 @@ However, ROS 2 only supports [TF2](https://wiki.ros.org/tf2) (not to be confused
 Quoting from TF (Tully Foote) himself on [ROS Answers](https://answers.ros.org/question/373068/ros2-foxy-tftransformationsquaternion_from_euler-equivalent/),
 > tf.transformations is a fork of [https://github.com/cgohlke/transformations/](https://github.com/cgohlke/transformations/). This package has been deprecated "Transformations.py is no longer actively developed and has a few known issues and numerical instabilities."
 
-The recommended alternative is a package available via `pip` called [`transforms3d`](https://matthew-brett.github.io/transforms3d/).
+The recommended alternative is a package called [`transforms3d`](https://matthew-brett.github.io/transforms3d/). This was originally only available via `pip` but is now available via `apt` and other package managers [in the `rosdistro`](https://github.com/ros/rosdistro/pull/33091).
 
 However, using that library has a few obstacles that make porting ROS 1 code to ROS 2 difficult.
  1. The API is different. The new API has more consistent naming, but even then, it is not a one-to-one translation. for example, `tf.transformations.quaternion_from_euler` could be replaced with `transforms3d.euler.euler2quat`, but `tf` returns the quaternion with the ordering `x, y, z, w` and `transforms3d` returns `w, x, y, z`.
  2. Not all of the functions have been migrated. As noted in the [`transforms3d` Refactoring Plan](https://matthew-brett.github.io/transforms3d/devel/refactor_plan.html), some of the functions are still in the "to integrate" pile.
 
 ## Migration
 If you're here, its likely because you want to migrate some code easily. You have two options:
- 1. Use `transforms3d` by adding a dependency on `python-transforms3d-pip` in your `package.xml` and noting differences in API discussed above.
+ 1. Use `transforms3d` by adding a dependency on `python3-transforms3d` in your `package.xml` and noting differences in API discussed above.
```

Comment:
This dependency is not required on the external package. It should come in depending only on this package.